helmfile.d/values/networkpolicies/common/fluentd.yaml.gotmpl

```
@@ -0,0 +1,43 @@
+{{- $netpol := .Values.networkPolicies -}}
+
+policies:
+  fluentd-system:
+    all-deny:
+      podSelectorLabels: {}
+      egress: {}
+      ingress: {}
+    forwarder:
+      podSelectorLabels:
+        app.kubernetes.io/instance: fluentd-forwarder
+      egress:
+        - rule: egress-rule-apiserver
+        - rule: egress-rule-dns
+        - name: fluentd-aggregator
+          peers:
+            - podSelectorLabels:
+                app.kubernetes.io/instance: fluentd-aggregator
+          ports:
+            - tcp: 24224
+      ingress:
+        - rule: ingress-rule-prometheus
+          ports:
+            - tcp: 24231
+    aggregator:
```

Style:
```suggestion
            - tcp: 24231
            
    aggregator:
```
I'd also prefer if the aggregator policy was above the forwarder.